bot.py

```diff
@@ -0,0 +1,214 @@
+import os
+import re
+import sys
+from datetime import datetime
+import time
+
+import discord
+import dotenv
+import redis
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_slash.utils.manage_commands import create_option, create_choice
+from dotenv import dotenv_values
+
+r = redis.Redis(host=dotenv_values("config.env")["HOSTNAME"], port=dotenv_values("config.env")["PORT"], db=dotenv_values("config.env")["DB"])
+
+now = datetime.now()  # current date and time
+date_time = now.strftime("%m/%d/%Y, %H:%M:%S")
+
+whitelisted = dotenv_values("config.env")["whitelisted"].split(",")
+bot = commands.Bot(command_prefix="redis ")
+slash = SlashCommand(bot, override_type=True, sync_commands=True)
+
+guild_ids = [int(dotenv_values("config.env")["guild_id"])]
+
+
+async def env_adj(key, value):
+    dotenv.set_key("config.env", str(key).upper(), value)
+
+
+@slash.slash(name="Reboot", description=f"", guild_ids=guild_ids)
+async def _reboot(ctx):
+    now = datetime.now()  # current date and time
+    result = discord.Embed(
+        title='Rebooting',
+        description=f'Execution time: {now}\n\n**ATTEMPTING REBOOT**',
+        colour=discord.Colour.teal()
```

Comment:
Try exporting texts into a single 'texts' module with keys, it will be easier to edit and maintain  (and add translations in the future)